versions/1.21/gradle.properties

```diff
@@ -16,19 +16,19 @@
 
 	# https://masa.dy.fi/maven/fi/dy/masa/malilib/
 	# https://jitpack.io/#sakura-ryoko/malilib
-	malilib_version = 1.21-sakura.2
+	malilib_version = 1.21-sakura.7
```

Comment:
The mc1.21 dev client fails to start, due to conflictions exist in the declared dependencies here:

- item scroller `1.21-sakura.4` conflicts with malilib `1.21-sakura.7`
- implicit older malilib dependency propagation from minihud

I'll suggested to just revert all changes from this file, as it's unrelated to this PR